Contribute/hacktoberfest.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Hacktoberfest and Microsoft docs
+description: Learn how to discover which docs repositories participate in Hacktoberfest, how to contribute, and what you can expect as a contributor.
+author: IEvangelist
+ms.author: dapine
+ms.date: 09/10/2021
+ms.topic: contributor-guide
+ms.prod: non-product-specific
+ms.custom: external-contributor-guide
+---
+
+# Hacktoberfest and Microsoft docs
+
+Hacktoberfest is an annual world-wide event that drives open source contributions through the month of October. During the month, open source developers are encouraged to contribute to repositories through pull requests (PRs). There are many Microsoft docs open source repositories on GitHub, all of which contribute to the [docs.microsoft.com](https://docs.microsoft.com) content. Some of the repositories actively participate in the Hacktoberfest event. In this article you'll learn how to discover which repos are accepting pull requests, and what you can expect as a contributor.

```

Comment:
```suggestion
Hacktoberfest is an annual worldwide event held during the month of October. The event encourages open source developers to contribute to repositories through pull requests (PR). GitHub hosts many Microsoft Docs open source repositories, all of which contribute to the [docs.microsoft.com](https://docs.microsoft.com) content. Some of the repositories actively participate in the Hacktoberfest event. In this article, you'll learn how to discover which repos are accepting PRs, and what you can expect as a contributor.

```

The PR acronym wasn't used in the rest of the article! I changed all references to "pull request" so that it is. :)